src/Header/Header.js

```diff
@@ -27,39 +27,46 @@ class Header extends Component {
 
   handleIconClick = () => {
     this.setState({ searchHidden: false })
-    this.state.ref.current.focus()
+    this.state.searchInput.current.focus()
   }
 
   render() {
-    const searchBar = <div className="input-container">
-    <i 
-      className={this.state.searchHidden ? `material-symbols-outlined` :
-        `material-symbols-outlined icon-transition`}
-      onClick={this.handleIconClick}
-      onKeyDown={(e) => this.handleKeyDown(e)}
-      tabIndex={1}
-    >search</i>
-    <input 
-      className={this.state.searchHidden ? `search-input` :
-      `search-input input-transition`}
-      type="search" 
-      name="search"
-      placeholder="search by title" 
-      value={this.state.search}
-      tabIndex={1}
-      onChange={(e) => this.handleChange(e)}
-      ref={this.state.ref}
-    />
+    const searchBar = 
+    <div className="input-container">
+      <i 
+        className={this.state.searchHidden ? `material-symbols-outlined` :
+          `material-symbols-outlined icon-transition`}
+        onClick={this.handleIconClick}
+        onKeyDown={(e) => this.handleKeyDown(e)}
+        tabIndex={1}
+      >search</i>
```

Comment:
Curious about the use of the <i> element here.  What led you to reach for that element?